packages/schema/src/remove.rs

```diff
@@ -37,7 +37,7 @@ pub fn remove_schemas(schemas_dir: &path::Path) -> Result<(), io::Error> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use std::ffi::OsStr;
+    // use std::ffi::OsStr;
```

Comment:
Why this change? Seems unrelatet and causes compile errors.

Reply:
that was a formatting thing, my bad gonna revert that. 

another question... and this is a stupid rust beginner question but since im on windows does 
```rust 
let source_ptr = write_data(&env, &[61; 33]);
```
the 33 here needs to be increased, not the 61 right? Testing locally returns the windows contract error